var/spack/repos/builtin/packages/py-datalad/package.py

```diff
@@ -34,6 +35,8 @@ class PyDatalad(PythonPackage):
 
     depends_on('python@3.6:', type=('build', 'run'))
     depends_on('py-setuptools@40.8.0:', type='build')
+    depends_on('py-packaging', type=('build', 'run'), when='@0.15.4:')
```

Comment:
Can you move this to the "core" section?

Reply:
I have put it to the build dependencies because it is listed in [`pyproject.toml`](https://github.com/datalad/datalad/blob/0.15.5/pyproject.toml#L3). But sure, I can also move it.

Reply:
Done